pulp_container/app/token_verification.py

```diff
@@ -9,6 +11,7 @@
 from rest_framework.exceptions import AuthenticationFailed, NotAuthenticated
 from rest_framework.permissions import BasePermission, SAFE_METHODS
 
+Scope = namedtuple("Scope", "resource_type, name, action")
```

Comment:
```suggestion
Scope = namedtuple("Scope", ["resource_type", "name", "action"])
```

Not blocking this, if it works, but it looks weird to me.

Reply:
@mdellweg, I was inspired by https://github.com/pulp/pulp_container/pull/217#discussion_r571015874. And I liked it.